scalding-core/src/main/scala/com/twitter/scalding/typed/cascading_backend/CascadingBackend.scala

```diff
@@ -25,9 +25,13 @@ import com.twitter.scalding.serialization.{
 }
 import java.util.WeakHashMap
 import scala.collection.mutable.{ Map => MMap }
+import org.apache.hadoop.mapred.JobConf
+import com.twitter.scalding.quotation.Quoted
 
 object CascadingBackend {
   import TypedPipe._
+  
+  private implicit val q: Quoted = Quoted.internal
```

Comment:
I'm nervous about this hanging out implicitly. Can we not use it implicitly and instead explicitly call it: `interalQuoted` where needed? I fear this will actually destroy our `Quoted` information when we are doing composing operations, and defeat the whole purpose.

Reply:
fixed